### ISSUE TITLE
Property "items" missing in FieldRenderProps

### ISSUE THREAD

**Issue by dininski** [final-form/react-final-form#398]
### Are you submitting a **bug report** or a **feature request**?
**bug report**

### What is the current behavior?

Property "items" missing in FieldRenderProps. When implementing a type-ahead component and using TypeScript, TS complains that the field `items` is missing in FieldRenderProps. The field component seems to have a `  [otherProp: string]: any` property, so it does not error out with `items` as a prop. However, because it extends `RenderableProps<FieldRenderProps>`, which has the following signature:

```typescript
export interface RenderableProps<FieldRenderProps> {
  children?: ((props: FieldRenderProps) => React.ReactNode) | React.ReactNode
  component?: React.ComponentType<FieldRenderProps> | string
  render?: (props: T) => React.ReactNode
}
```
the build errors out and the TS error needs to be ignored.

### What is the expected behavior?

The `items` field should be present in the `FieldRenderProps`, albeit optional. Or there should be an overload for this specific use case.

### Sandbox Link

Unfortunately the issue does not present itself within codesandbox,io

### What's your environment?

A TypeScript React application, bootstrapped with `create-react-app`. TS version: 3.2.2, react-final-form: 4.0.2, final-form: 4.11.0

### Other information

Exact error:
```
Error:(44, 31) TS2339: Property 'items' does not exist on type 'Readonly<{ children?: ReactNode; }> & Readonly<FieldRenderProps>'.
```

**Comment by iamandrewluca** [comment]
https://github.com/final-form/react-final-form/issues/175

**Comment by myagoo** [comment]
Same issue here

**Comment by erikras** [comment]
Published fix in [`v6.3.4`](https://github.com/final-form/react-final-form/releases/tag/v6.3.4).
